EIPS/eip-FOCIL.md

```diff
@@ -147,11 +146,11 @@ class SignedLocalInclusionList(Container):
 ## Rationale
 
 ### Core Properties
-- Committee-based: FOCIL relies on a committee of multiple validators, rather than a single proposer, to construct and broadcast local ILs. This approach significantly reduces the surface for bribery and extortion attacks and strengthens censorship resistance.
-- Fork-choice enforced: FOCIL incorporates the force-inclusion mechanism into the fork-choice rule, an integral component of the consensus process, thereby preventing any actor from bypassing the system. Attesters vote only for blocks that include transactions from a set of local ILs provided by the IL committee and that satisfy the IL constraints. Any block failing to meet these criteria is deemed invalid.
+- Committee-based: FOCIL relies on a committee of multiple validators, rather than a single proposer, to construct and broadcast ILs. This approach significantly reduces the surface for bribery and extortion attacks and strengthens censorship resistance.
+- Fork-choice enforced: FOCIL incorporates the force-inclusion mechanism into the fork-choice rule, an integral component of the consensus process, thereby preventing any actor from bypassing the system. Attesters vote only for blocks that include transactions from a set of ILs provided by the IL committee and that satisfy the IL constraints. Any block failing to meet these criteria is deemed invalid.
```

Comment:
`invalid` is a strong here. I'd suggest changing it to "Any block failing to meet these criteria will not be voted on by the attesters, and therefore cannot be canonical."